Onion/Controllers/LoggerController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Onion.Core.Interfaces.Services;
+using Onion.Core.Models;
+using Onion.Models;
+
+namespace Onion.Controllers
+{
+    public class LoggerController : Controller
+    {
+        private readonly ILoggerService _loggerService;
+
+        public LoggerController(ILoggerService loggerService)
+        {
+            _loggerService = loggerService;
+        }
+
+        public IActionResult Index(string id)
```

Comment:
Can we just name this to be more descriptive guys. The code all looks great. Yet I like the method from "Clean Code" where you create a story in your code. In this case I would have likely named the controller something like ViewLogsController. The method on the loggerService GetLogErrorbyModelID(modelID). It's good to be overly descriptive.

Reply:
Working on it